### ISSUE TITLE
Decide what kind of email title to use

### ISSUE THREAD

**Issue by juho-jaakkola** [AU-Landing-Project/comment_tracker#15]
Starting from Elgg 1.9 the title of comment notifications is by default of form:

```RE: <title of the commented object>```

Currently this overrides comment_tracker's custom titles like:
```php
'comment:notify:subject:blog' => '%s commented on the blog "%s"',
```

Options:
 1. Remove custom titles and always use Elgg's default title
 2. Add handler for [email, system] on low priority and add the custom titles there

**Comment by juho-jaakkola** [comment]
I was wrong. The title is actually being added by the Groups plugin and not the notifications system. This is an inconsistency in Elgg and should be fixed there before making the decision for comment_tracker.

**Comment by juho-jaakkola** [comment]
https://github.com/Elgg/Elgg/issues/7743

**Comment by juho-jaakkola** [comment]
Fixed in a4a8da3fafb95f0f5a37f4e750a1fa9760110bef
